app/src/main/java/ch/epfl/favo/viewmodel/IFavorViewModel.java

```diff
@@ -1,46 +1,52 @@
-package ch.epfl.favo.viewmodel;

```

Comment:
Are you reformatting the code? For some reason it doesn't track changes properly

Reply:
No, I don't think I touched this.